gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,68 @@ Features marked with `PREVIEW` require a GitKraken Account, with access level ba
 
 ---
 
+<a id="v16-2"></a>
+
+## Version 16.2
+
+#### Thursday, January 16, 2025
+
+GitLens 16.2 brings support for self-hosted GitHub Enterprise and GitLab Self-Managed integrations, introduces expanded merge target status tracking, and improves search and navigation features in Launchpad and the Home View. New visualizations for folder history and improved merge request search make it easier to track changes and manage Git workflows.
+
+<img src="/wp-content/uploads/gl-16-2-hero.png" class="img-responsive center img-bordered">
+
+### Self-Hosted GitHub and GitLab Integration Support `PRO`
+GitLens Pro now supports integration with self-hosted GitHub Enterprise and GitLab Self-Managed instances, allowing users to access issue, pull request, and merge request context directly within GitLens.
+
+- Issues and pull/merge requests from GitHub Enterprise and GitLab Self-Managed will now appear in the GitLens [Home View](https://help.gitkraken.com/gitlens/home-view/) when linked to branches.
+- The [Start Work](https://help.gitkraken.com/gitlens/home-view/#starting-work-on-an-issue) flow now allows users to create branches and worktrees from issues in GitHub Enterprise and GitLab Self-Managed.
```

Comment:
So far all we have managed to get working is PRs for GitHub Enterprise in Launchpad. We haven't been able to add GitLab Self Managed because we have no way to test our changes and we haven't connected anything to Start Work yet.

We may need to modify this messaging prior to release.

Reply:
I'll update this depending on what we're actually shipping.

Reply:
Looks like we managed to get this in, so marking this resolved.